discovery_engine_core/web-api/compose.yml

```diff
@@ -1,42 +0,0 @@
-services:
```

Comment:
❓ the docker files were pretty useful to setup local testing environments, shouldn't they be updated as well (where appropriate) instead of deleting them?

Reply:
I moved them over, I will update them in

- #691